Mathlib/Data/Fin/Tuple/Basic.lean

```diff
@@ -444,8 +446,17 @@ theorem repeat_add {α : Type*} (a : Fin n → α) (m₁ m₂ : ℕ) : Fin.repea
   · simp [modNat, Nat.add_mod]
 #align fin.repeat_add Fin.repeat_add
 
-proof_wanted repeat_comp_rev {α} (a : Fin n → α) :
-  (Fin.repeat m a) ∘ Fin.rev = Fin.repeat m (a ∘ Fin.rev)
+theorem repeat_comp_rev {α} (a : Fin n → α) :
+    (Fin.repeat m a) ∘ Fin.rev = Fin.repeat m (a ∘ Fin.rev) := by
+  ext ⟨j, h₁⟩
+  simp [rev, modNat]
```

Comment:
Non-closing `simp`s aren't allowed in Mathlib. Please use `simp?` to turn it into `simp only` (or into` simp? [] says ...`).

Reply:
Fixed.